src/jit/lclvars.cpp

```diff
@@ -994,6 +950,10 @@ void Compiler::lvaInitUserArgs(InitVarDscInfo* varDscInfo)
             varDscInfo->setAllRegArgUsed(argType);
 
 #endif // _TARGET_XXX_
+
+#ifdef FEATURE_FASTTAILCALL
```

Comment:
All cases of `#ifdef FEATURE_FASTTAILCALL` need to be replaced with `#if FEATURE_FASTTAILCALL`